packages/react-components/react-table/src/components/TableCellLayout/TableCellLayout.types.ts

```diff
@@ -3,15 +3,23 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 export type TableCellLayoutSlots = {
   root: Slot<'div'>;
 
-  media?: Slot<'span'>;
+  media: Slot<'span'>;
+
+  main: Slot<'span'>;
+
+  description: Slot<'span'>;
+
+  wrapper: Slot<'div'>;
```

Comment:
nit: Would be nice to have some comments here

Reply:
Added